manifests/params.pp

```diff
@@ -106,6 +106,37 @@
                                   '/usr/local/share/syslinux/bios/com32/mboot/mboot.c32',
                                   '/usr/local/share/syslinux/bios/com32/menu/menu.c32']
     }
+    'Archlinux': {
+      # if set to true, no repo will be added by this module, letting you to
+      # set it to some custom location.
+      $custom_repo         = true # TODO: no packages available yet in AUR
+      $plugin_prefix       = 'ruby-smart-proxy-' # TODO: no plugins available in AUR yet
+
+      $dir   = '/usr/share/foreman-proxy'
+      $etc   = '/etc'
+      $shell = '/usr/bin/false'
+      $user  = 'foreman-proxy'
+      $puppet_home = '/var/lib/puppet'
+      $puppet_bindir = '/usr/bin'
+      $puppetdir = '/etc/puppetlabs/puppet'
+      $ssldir = "${puppetdir}/ssl"
+
+      $puppetssh_command = "${puppet_bindir}/puppet agent --onetime --no-usecacheonfailure"
```

Comment:
Opened https://github.com/theforeman/puppet-foreman_proxy/issues/309 to track this